src/lib/layout/Footer.svelte

```diff
@@ -0,0 +1,95 @@
+<script>
+  import Icon from 'svelte-icons-pack/Icon.svelte';
+  import FaBrandsInstagram from 'svelte-icons-pack/fa/FaBrandsInstagram';
+  import FaBrandsTwitter from 'svelte-icons-pack/fa/FaBrandsTwitter';
+  import FaBrandsFacebook from 'svelte-icons-pack/fa/FaBrandsFacebook';
+  import FaBrandsGithub from 'svelte-icons-pack/fa/FaBrandsGithub';
+  import IoMail from 'svelte-icons-pack/io/IoMail';
+  import FaBrandsLinkedin from 'svelte-icons-pack/fa/FaBrandsLinkedin';
+  import FaSolidUser from 'svelte-icons-pack/fa/FaSolidUser';
+</script>
+
+<div class="w-full bg-transparent p-3 max-sm:hidden z-10">
+  <footer class="p-2 text-white grid grid-cols-3 justify-between border-t-2 border-secondary">
+    <div class="flex p-3 self-center gap-3">
+      <a href="https://www.instagram.com/niaefeup/"
+        ><Icon src={FaBrandsInstagram} color="white" size="24px" /></a
+      >
+      <a href="https://twitter.com/niaefeup"
+        ><Icon src={FaBrandsTwitter} color="white" size="24px" /></a
+      >
+      <a href="https://www.facebook.com/NIAEFEUP"
+        ><Icon src={FaBrandsFacebook} color="white" size="24px" /></a
+      >
+      <a href="https://github.com/NIAEFEUP"
+        ><Icon src={FaBrandsGithub} color="white" size="24px" /></a
+      >
+      <a href="https://www.linkedin.com/company/nifeup"
+        ><Icon src={FaBrandsLinkedin} color="white" size="24px" /></a
+      >
```

Comment:
I have moved Icons into a new folder in the `lib` folder, `component`. There are two files, `Icon.svelte` is a component that takes in the information of the icon as well as if it should have an anchor tag with a link; `Icon.ts` is a simple typescript file that imports all of the icons we are using and places them in an object that is then exported and used wherever the icons are needed, alongside the previous component. This solution has made it so that creating icons with links is easier and has uncluttered the imports of files using them, however adding  new icon is a tad more cumbersome, as we must import it and then add it to the object. Any suggestions on improving it?

Reply:
I think the current implementation is OK. Would love to hear second other opinions if anyone has one